public/scripts/controllers/playController.js

```diff
@@ -0,0 +1,122 @@
+angular.module('scrambleApp').
+  controller('playController', ['$scope', '$interval', 'wordFactory', 'gameStateService',
+             function($scope, $interval, wordFactory, gameStateService) {
+    //Controls the gameplay part
+
+    $scope.letters = [];
+    $scope.userInput = [];
+    $scope.score = 0;
+
+    var inputReady = true;
+    var currentWord;
+    var numCharacters; //number of characters
+    var startTime; //marks when word was given to give score
+    var endTime = Date.now() + 60999; //set end time to 60s from current time
+    var timer;
+
+    //Set scope 'letters' to a new random word
+    var getRandomWord = function() {
+      inputReady = false;
+      wordFactory.getRandomWord(function(randomWord, error) {
+        if (error) {
+          console.log(error);
+        } else {
+          currentWord = randomWord;
+          var shuffledWord = wordFactory.shuffleWord(currentWord);
+          $scope.letters = shuffledWord.map(function(letter) {return {letter:letter}});
+          $scope.userInput = [];
+          startTime = Date.now();
+          numCharacters = shuffledWord.length;
+        }
+        inputReady = true;
+      });
+    };
+
+    //Sets timer to the difference between current time and the end time
+    var updateTimer = function() {
+      $scope.timer = Math.floor((endTime - Date.now())/1000);
+      if ($scope.timer <= 0) {
+        //Trigger the end game
+        console.log(currentWord);
+        gameStateService.setState({
```

Comment:
the setState call seems to be doing a couple of things, so you probably want to split it up to separate calls (I don't see a reason to make this atomic)